lib/xandra/authenticator/password.ex

```diff
@@ -0,0 +1,7 @@
+defmodule Xandra.Authenticator.Password do
+  @behaviour Xandra.Authenticator
+
+  def response_body([username: username, password: password]) do
+    <<0x00>> <> username <> <<0x00>> <> password
```

Comment:
Since we should use iodata for this API, this can be written as

```elixir
def response_body(options) do
  [0x00, Keyword.fetch!(options, :username), 0x00, Keyword.fetch!(options, :password)]
end
```